### PR TITLE
chore(deps): update `canvas` to v2.11.2

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview

Refs https://github.com/votingworks/vxsuite/issues/4163

v2.9.1 is incompatible with NodeJS v20. Also, v2.11.2 is incompatible with NodeJS v22, but that's a problem for future us.

## Demo Video or Screenshot
n/a

## Testing Plan
Automated tests, plus I tested manually building an image using `createCanvas` and it worked:

test

```js
const canvas = require("canvas").createCanvas(100, 100);
const ctx = canvas.getContext("2d");
ctx.fillStyle = "red";
ctx.fillRect(20, 20, 40, 40);
const png = canvas.toBuffer("image/png");
fs.writeFileSync("test.png", png);
```

### CODE REVIEW

File: libs/ballot-interpreter/src/hmpb-ts/interpret.test.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { assertDefined, iter, ok, unique } from '@votingworks/basics';
+import { ImageData } from 'canvas';
```
Comment (adghayes) [inline libs/ballot-interpreter/src/hmpb-ts/interpret.test.ts:2]: Why did you need to add in all these type imports?

**Review by adghayes — APPROVED** [review]

File: libs/ballot-interpreter/src/hmpb-ts/interpret.test.ts (hunk 1 above)
Comment (eventualbuddha) [reply libs/ballot-interpreter/src/hmpb-ts/interpret.test.ts:2]: The types of `canvas.ImageData` and `globalThis.ImageData` used to be compatible but no longer are:

```ts
// canvas@2.11.2
export class ImageData {
	constructor(sw: number, sh: number);
	constructor(data: Uint8ClampedArray, sw: number, sh?: number);
	readonly data: Uint8ClampedArray;
	readonly height: number;
	readonly width: number;
}

// lib.dom.d.ts
/**
 * The underlying pixel data of an area of a <canvas> element. It is created using the ImageData() constructor or creator methods on the CanvasRenderingContext2D object associated with a canvas: createImageData() and getImageData(). It can also be used to set a part of the canvas by using putImageData().
 *
 * [MDN Reference](https://developer.mozilla.org/docs/Web/API/ImageData)
 */
interface ImageData {
    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ImageData/colorSpace) */
    readonly colorSpace: PredefinedColorSpace;
    /**
     * Returns the one-dimensional array containing the data in RGBA order, as integers in the range 0 to 255.
     *
     * [MDN Reference](https://developer.mozilla.org/docs/Web/API/ImageData/data)
     */
    readonly data: Uint8ClampedArray;
    /**
     * Returns the actual dimensions of the data in the ImageData object, in pixels.
     *
     * [MDN Reference](https://developer.mozilla.org/docs/Web/API/ImageData/height)
     */
    readonly height: number;
    /**
     * Returns the actual dimensions of the data in the ImageData object, in pixels.
     *
     * [MDN Reference](https://developer.mozilla.org/docs/Web/API/ImageData/width)
     */
    readonly width: number;
}
```

Notably, `colorSpace` is present in the DOM definition but not in the `canvas` one. So to avoid type errors we always use the one from `canvas`.
